lib/repl.js

```diff
@@ -231,7 +231,7 @@ function REPLServer(prompt,
       try {
         try {
           const scriptOptions = {
-            displayErrors: true,
+            displayErrors: false,
```

Comment:
Any chance you could split the `displayErrors` changes out into a separate commit?